[MM-29853] Migrate string-refs to functional ones: this.refs.fileInput

lestgabo opened the pull request:
#### Summary
Update this.refs.fileInput to this.fileInputRef react ref function

#### Ticket Link
github issue ->Fixes https://github.com/mattermost/mattermost-server/issues/16036
jira -> https://mattermost.atlassian.net/browse/MM-29853

#### Related Pull Requests
n/a

#### Screenshots
n/a

mattermod commented:
Hello @lestgabo,

Thanks for your pull request! A Core Committer will review your pull request soon. For code contributions, you can learn more about the review process [here](https://developers.mattermost.com/contribute/getting-started/code-review/).

nickmisasi reviewed (approved):
Looks great! Thanks @lestgabo!

mattermod commented:
This PR has been automatically labelled "stale" because it hasn't had recent activity.
A core team member will check in on the status of the PR to help with questions.
Thank you for your contribution!

/cc @jasonblais @jfrerich @emilyacook

mattermod commented:
This PR has been automatically labelled "stale" because it hasn't had recent activity.
A core team member will check in on the status of the PR to help with questions.
Thank you for your contribution!

/cc @jasonblais @jfrerich @emilyacook

jgilliam17 commented:
/update-branch

mattermod commented:
Error trying to update the PR.
Please do it manually.

hmhealey commented:
@lestgabo Have you renamed or deleted the branch for this? We've tried to update the branch to the latest master, but it seems to be failing because the branch moved.

lestgabo commented:
@hmhealey aahh sorry, i redid my fork to be https://github.com/lestgabo/mattermost-webapp-1
I didnt realize this one would pose problems as all the CI checks were green before I deleted my fork.

I already deleted my mattermost folder and it would take me some time to set it up again just to redo this. I made a mess of the commits on this PR by not rebasing and not updating the fork. This was my bad! 

Please go ahead and close this PR without merging.

hmhealey commented:
No worries. We can copy the commits to a new branch and resubmit them. You'll still be credited on them 😄
